flocker/acceptance/endtoend/test_dataset.py

```diff
@@ -10,6 +10,7 @@
 from twisted.trial.unittest import TestCase
 
 from ...common import loop_until
+from ...testtools import flaky
 
 from ..testtools import (
     require_cluster, require_moving_backend, create_dataset, DatasetBackend
```

Comment:
I think my point was more that it's even more confusing than you realized. :grinning: